### PR TITLE
feat: specification guides

### PR DESCRIPTION
**Pull request by oliviergoulet5**
Resolves part of #1847 

These won't be linked anywhere until the redesigned learn page is live.

### CODE REVIEW

File: content/learn/specification-guides/concurrency-explained.md
Comment (oliviergoulet5) [inline content/learn/specification-guides/concurrency-explained.md]: These quotes are the Unicode ones. I need to replace `“` to `"` across all files

**Review by ericpoirier — APPROVED** [review]

**Review by chrisguindon — APPROVED** [review]

**Comment by chrisguindon** [comment]
> These won't be linked anywhere until the redesigned learn page is live.

Sure but they should start to show up in google searches because of sitemap.xml.

We should make sure the client is aware of that caveat to make sure they are okay with this before merging...

**Comment by oliviergoulet5** [comment]
> We should make sure the client is aware of that caveat to make sure they are okay with this before merging...

They are now aware of it and have given their approval.

**Pull request merged**
